[2022-08-23 20:52:19,639] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: gmailPipeline.gmailAuthTask manual__2022-08-24T00:52:17.629505+00:00 [queued]>
[2022-08-23 20:52:19,644] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: gmailPipeline.gmailAuthTask manual__2022-08-24T00:52:17.629505+00:00 [queued]>
[2022-08-23 20:52:19,644] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 20:52:19,644] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-23 20:52:19,644] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 20:52:19,654] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): gmailAuthTask> on 2022-08-24 00:52:17.629505+00:00
[2022-08-23 20:52:19,656] {standard_task_runner.py:52} INFO - Started process 6411 to run task
[2022-08-23 20:52:19,661] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'gmailPipeline', 'gmailAuthTask', 'manual__2022-08-24T00:52:17.629505+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/gmailDag.py', '--cfg-path', '/var/folders/mm/xv24gcjj46l_jx1m76p5bgy80000gn/T/tmptgbnlmrw', '--error-file', '/var/folders/mm/xv24gcjj46l_jx1m76p5bgy80000gn/T/tmpqlz7zinb']
[2022-08-23 20:52:19,663] {standard_task_runner.py:80} INFO - Job 12: Subtask gmailAuthTask
[2022-08-23 20:52:19,714] {task_command.py:371} INFO - Running <TaskInstance: gmailPipeline.gmailAuthTask manual__2022-08-24T00:52:17.629505+00:00 [running]> on host abhays-mbp.lan
[2022-08-23 20:52:19,763] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=gmailPipeline
AIRFLOW_CTX_TASK_ID=gmailAuthTask
AIRFLOW_CTX_EXECUTION_DATE=2022-08-24T00:52:17.629505+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-24T00:52:17.629505+00:00
[2022-08-23 20:52:19,893] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGABRT
[2022-08-23 20:52:19,910] {taskinstance.py:1415} INFO - Marking task as FAILED. dag_id=gmailPipeline, task_id=gmailAuthTask, execution_date=20220824T005217, start_date=20220824T005219, end_date=20220824T005219
[2022-08-23 20:52:19,934] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
